src/js/providers/utils/quality-labels.ts

```diff
@@ -32,29 +44,31 @@ export function createLabel(height, bandwidth, redundant) {
 
 // Ensures that we're able to find a custom label. As long as there is at least 1 quality label and a defined
 // bandwidth, a quality label will always be found. Return null otherwise
-export function getCustomLabel(qualityLabels, bandwidth) {
-    let label = null;
+export function getCustomLabel(qualityLabels: QualityLabels | null, bandwidth?: number): string | null {
```

Comment:
This function is _only_ used in `generateLabel` so there is no reason to export it.

`qualityLabels` could be made optional (it could be `undefined`, not `null`). Nothing in the config defaults it to `null`.

Reply:
> This function is _only_ used in `generateLabel` so there is no reason to export it.
> 
Exported to be unit tested as well.

Reply:
Does it need to be? It’s a sub-function of one other function so it’s only split out for organization/readability. Ideally it would be inlined and all tests would be on the parent function that we actually use and the inputs and outputs we expect from it.

Reply:
Does it need to be? It’s a sub-function of one other function so it’s only split out for organization/readability. Ideally it would be inlined and all tests would be on the parent function that we actually use and the inputs and outputs we expect from it.